gen/aws/templates/cloudformation.json

```diff
@@ -71,10 +71,10 @@
         "stable": "ami-05bc0164"
       },
       "us-west-1": {
-        "stable": "ami-27553a47"
```

Comment:
For DC/OS overlay we need docker 1.9 and higher. The cloud formation template has an older CoreOS AMI that comes with Docker 1.7. So in order to test the overlay features with cloud formation, we needed a newer AMI. 

I will move this to a separate commit so that it can be discarded in case we plan to upgrade the AMI using a separate commit.